odl/tomo/backends/astra_setup.py

```diff
@@ -339,7 +347,7 @@ def astra_projection_geometry(geometry):
             'parallel3d', det_width, det_height, det_row_count,
             det_col_count, angles)
 
-    elif isinstance(geometry, CircularConeFlatGeometry):
+    elif False and isinstance(geometry, CircularConeFlatGeometry):
```

Comment:
If we *really* need FDK then we need a clause here. But that's another issue and not the topic here. Parallel beam is a different story again. I'd say delete it here.